exercises/two-fer/src/main/kotlin/Twofer.kt

```diff
@@ -0,0 +1,3 @@
+internal fun twofer(name: String): String {
+    return ""
```

Comment:
Instead, I suggest throwing a [not implemented error](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-not-implemented-error/index.html).